actions/settings.py

```diff
@@ -141,4 +144,30 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+
+# CSP
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_REPORT_ONLY = DEBUG
+CSP_DEFAULT_SRC = ["'self'"]
+CSP_FONT_SRC = ["'self'", "https://fonts.gstatic.com"]
+CSP_SCRIPT_SRC_ELEM = ["'self'"]
+CSP_STYLE_SRC_ELEM = ["'self'", "https://fonts.googleapis.com"]
+
+# which directives to set a nonce for
+CSP_INCLUDE_NONCE_IN = ["script-src", "script-src-elem"]
+
+# configure django-csp to work with Vite when using it in dev mode
+if DJANGO_VITE_DEV_MODE:  # pragma: no cover
+    CSP_CONNECT_SRC = ["ws://localhost:3000/static/"]
+    CSP_STYLE_SRC_ELEM = ["'self'", "https://fonts.googleapis.com", "'unsafe-inline'"]
+    CSP_SCRIPT_SRC_ELEM = ["http://localhost:3000"]
+
+
+# Permissions Policy
+# https://github.com/adamchainz/django-permissions-policy/blob/main/README.rst
+PERMISSIONS_POLICY = {
```

Comment:
Does this deny anything not listed? It's not clear from the docs, but it does have an example policy to disable most things, which suggests that might not be the case?

Reply:
No, everything is allowed by default.  Setting `<key>: []` disallows it though.

Reply:
Are we going with a minimal policy to start, with the intention of tightening it up in future? I think it makes sense to do that, I just wanted to understand!

Reply:
Kind of!  I originally added this to opt-out of Google's FLoC, which is [maybe] dead now.  However I wanted to keep the support of these headers around (and since there is still some support for the FLoC header knocking around), so stuck with this very minimal option to start.

Reply:
[FLoC](https://github.com/WICG/floc) has been replaced by the [Topics API](https://github.com/patcg-individual-drafts/topics).

Reply:
That was the name, thank you!

The Topics API proposal contains

> Note: The old Permissions-Policy: interest-cohort=() from FLoC will also forbid topic calculation.

(via [django-permissions-policy's changelog](https://github.com/adamchainz/django-permissions-policy/blob/main/HISTORY.rst#4110-2022-05-27))